### PR TITLE
Run hsts-prune and update impacted rulesets

### PR DESCRIPTION
**Pull request by Bisaloo**

### CODE REVIEW

File: src/chrome/content/rules/Marx.org.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+-->
+<ruleset name="Marx.org">
+
+	<target host="marx.org" />
+	<target host="www.marx.org" />
+
+	<rule from="^http://(www\.)?marx\.org/"
+		to="https://www.marxists.org/" />
+
+</ruleset>
```
Comment (pipboy96) [inline src/chrome/content/rules/Marx.org.xml]: Is it possible to remove this redirect?

File: src/chrome/content/rules/Nstatic.com.xml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<ruleset name="Nstatic.net">
+
+	<target host="nstatic.net" />
+
+	<rule from="^http:" to="https:" />
+
+</ruleset>
```
Comment (pipboy96) [inline src/chrome/content/rules/Nstatic.com.xml]: Are there any subdomains?

Comment (Bisaloo) [reply src/chrome/content/rules/Nstatic.com.xml]: I'd rather keep the changes in this PR as minimal as possible. Only run hsts-prune and fix the result. Enhancements can take place in future PRs

Comment (pipboy96) [reply src/chrome/content/rules/Nstatic.com.xml]: I will make such PR after this is merged.

File: src/chrome/content/rules/Marx.org.xml (hunk 1 above)
Comment (Bisaloo) [reply src/chrome/content/rules/Marx.org.xml]: I've removed this ruleset entirely as the redirect wasn't correct. See 17f3362

File: src/chrome/content/rules/Nstatic.com.xml (hunk 2 above)
Comment (ivysrono) [reply src/chrome/content/rules/Nstatic.com.xml]: merged.

**Pull request merged**